pom.xml

```
@@ -7,7 +7,7 @@
 
   <properties>
     <!-- set the version for camunda BPM here -->
-    <version.camunda>7.18.0</version.camunda>
+    <version.camunda>7.19.0-alpha5</version.camunda>
     <version.junit>4.13.2</version.junit>
     <version.assertj>3.21.0</version.assertj>
```

🙃 In some other examples, we moved assertj to 3.22.0.
```suggestion
    <version.assertj>3.22.0</version.assertj>
```